Sources/Commands/SwiftPackageTool.swift

```diff
@@ -625,6 +630,11 @@ extension SwiftPackageTool {
             print("Files written to", symbolGraphDirectory.pathString)
         }
     }
+    
+    enum ExtensionBlockSymbolBehavior: String, EnumerableFlag {
+        case emitExtensionBlockSymbols
+        case omitExtensionBlockSymbols
```

Comment:
should this be Boolean instead?

Reply:
AFAIK this is the only way to achieve a flag inversion with a custom name, i.e. with a prefix other than `enable`/`disable` or `no`. See the ArgumentParser  docs [here](https://apple.github.io/swift-argument-parser/documentation/argumentparser/declaringarguments#Using-flag-inversions-enumerations-and-counts):

> To create a flag with custom names for a Boolean value[...], define an enumeration that conforms to the EnumerableFlag protocol.

IMO the `emit`/`omit` prefixes describe the semantics best and I wouldn't want to have entirely different names for SwiftPM and compiler commands.

Reply:
ok